test/resources-responses.test.js

```diff
@@ -22,7 +22,25 @@ describe('Test Resources responses', function () {
     fixtures.disableScsbFixtures()
   })
 
-  describe('GET all_bibs', () => {
+  describe('GET all_items', () => {
+    it('reverts to filtering on inner_hits when item filters are included in the query', (done) => {
+      const url = global.TEST_BASE_URL + '/api/v0.1/discovery/resources/b10833141?all_items=true&item_status=status:i'
+      request.get(url, (err, res, body) => {
+        if (err) throw err
+        const doc = JSON.parse(body)
+        expect(doc.items.length).to.equal(41)
+        done()
+      })
+    })
+    it('reverts to filtering on inner_hits when item filters are included in the query and returns default item size', (done) => {
```

Comment:
this is not actually a documented requirement, but I think item filters should override all_items until we have the work done in discovery api to filter and return more than `items_size`  number of items